packages/components/src/Checkbox/CheckboxGroup.test.tsx

```diff
@@ -122,24 +93,22 @@ test('passes down "read-only" to checkboxes', () => {
 
 test('passes down "error" to checkboxes', () => {
   render(
-    <ThemeProvider theme={theme}>
-      <CheckboxGroup label="Group of Checkboxes" error>
-        <Checkbox value="one" data-testid="one">
-          one
-        </Checkbox>
-        <Checkbox value="two" data-testid="two">
-          two
-        </Checkbox>
-        <Checkbox value="three" data-testid="three">
-          three
-        </Checkbox>
-      </CheckboxGroup>
-    </ThemeProvider>
+    <CheckboxGroup label="Group of Checkboxes" error>
+      <Checkbox value="one" data-testid="one">
+        one
+      </Checkbox>
+      <Checkbox value="two" data-testid="two">
+        two
+      </Checkbox>
+      <Checkbox value="three" data-testid="three">
+        three
+      </Checkbox>
+    </CheckboxGroup>
   );
 
-  expect(screen.getByText('one')).toHaveClass(`data-[error]:bg-red-500`);
-  expect(screen.getByText('two')).toHaveClass(`data-[error]:bg-red-500`);
-  expect(screen.getByText('three')).toHaveClass(`data-[error]:bg-red-500`);
+  // expect(screen.getByText('one')).toHaveClass(`data-[error]:bg-red-500`);
```

Comment:
ehem :D

Reply:
Actually there's no red color for errors in `theme-core` 
Is that right  !

Reply:
there is one in colors.ts:
```
error: {
    text: '#f33',
    bg: '#f33',
  },
```

Reply:
But why should the `checkbox` have theses classes 
the red color should be for helper text not the `checkbox` itself

Reply:
so just remove the comments